internal/core/items.rs

```diff
@@ -162,6 +162,73 @@ pub struct ItemVTable {
     ) -> RenderingResult,
 }
 
+fn find_sibling_outside_repeater(
```

Comment:
I do not see many users for this but the two places it is used in here, so that's where I put them.

How about extracting itemrc + its helper cfunctions and classes from here into itemrc.rs? That would then contain the "wrapper code" plus the tree navigation, leavint items.rs for the actual implmentation.